attributes/default.rb

```diff
@@ -10,30 +10,33 @@
   default["sensu"]["windows"]["package_options"] = nil
   default["sensu"]["windows"]["install_dotnet"] = true
   default["sensu"]["windows"]["dotnet_major_version"] = 4
+  default["sensu"]["version"] = "1.0.2-1-x64"
+  default['sensu']['windows_package_version'] = '1.0.2.1'
 else
   default["sensu"]["admin_user"] = "root"
   default["sensu"]["directory"] = "/etc/sensu"
   default["sensu"]["log_directory"] = "/var/log/sensu"
+  default["sensu"]["version"] = "1.0.2-1"
```

Comment:
Is this required to use the new logrotate? I'd prefer to keep the changes as isolated as possible. I am not sure that we should default to 1.x yet as there are a handful of community plugins that have conflicts and need to be worked out first. You can always overwrite the version in your wrapper cookbook.

Reply:
Linux installation version does not come with a 32/64 bit version.

Previous it was all the same for both Windows and Linux.